docs/incentivizedtestnet/03-challenges/02-details/01-build_a_verifier.md

```diff
@@ -22,7 +22,11 @@ From higher to lower priority:
 * Stone
     * Objective: Support verification of Starkware zkRollup’s proofs as well as generic proofs generated via Cairo programs
 * SP1
+    * Compressed STARK; PLONK 
+* ZKM
+    * Plonky2; STARK
 * Jolt
+    * Lasso
```

Comment:
I was thinking more about a general sentence before the verifiers' list, something like:

"Please note that we are looking for verifiers of the original proving system, and not the ones used as wrappers on Ethereum (Groth16, fflonk, etc.) for on-chain optimization purposes"